src/sentry/api/endpoints/auth_index.py

```diff
@@ -109,9 +113,20 @@ def put(self, request: Request):
                 challenge = json.loads(validator.validated_data["challenge"])
                 response = json.loads(validator.validated_data["response"])
                 authenticated = interface.validate_response(request, challenge, response)
+                if not authenticated:
+                    logger.warning(
+                        "u2f_authentication.verification_failed",
```

Comment:
i imagine we'll probably want the user_id, how else would we be able to tell what user is having issues?

Reply:
can you think of other fields that might be useful?

Reply:
i can't think of any fields that is related to the authentication fn, since the challenge and response data wouldn't show anything useful on why it failed